.azurepipelines/Ubuntu-GCC5.yml

```diff
@@ -50,8 +50,8 @@ extends:
         Pkgs: 'PcAtChipsetPkg,ShellPkg,StandaloneMmPkg,BaseTools'
         Targets: 'DEBUG,RELEASE,NO-TARGET,NOOPT'
         ArchList: $(arch_list)
-      TARGET_TEST:
-        Pkgs: 'UnitTestFrameworkPkg'
+      TARGET_TEST_POLICY:

```

Comment:
This needs to be added to the Windows yml, too, right?

Reply:
oops

Reply:
funnily, we aren;t building UnitTestFrameworkPkg on windows for some reason..